### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by cristiancobo**

### CODE REVIEW

File: cypress/page/t-shirts/t-shirts.page.ts [hunk 1]
```diff
@@ -8,12 +8,19 @@ export class Tshirts{
       this.addToCartButton = '.ajax_add_to_cart_button.btn.btn-default';
     }
   
+
+    public findTShirtByName(tshirtName:string){
+
```
Comment (holgiosalos) [inline cypress/page/t-shirts/t-shirts.page.ts]: remove this blank line

File: cypress/page/t-shirts/t-shirts.page.ts [hunk 2]
```diff
@@ -8,12 +8,19 @@ export class Tshirts{
       this.addToCartButton = '.ajax_add_to_cart_button.btn.btn-default';
     }
   
+
+    public findTShirtByName(tshirtName:string){
+    
+      return cy.get(".product-name").filter(`[title="${tshirtName}"]`);
+
```
Comment (holgiosalos) [inline cypress/page/t-shirts/t-shirts.page.ts]: remove this blank line

File: cypress/page/t-shirts/t-shirts.page.ts [hunk 3]
```diff
@@ -8,12 +8,19 @@ export class Tshirts{
...
+      return cy.get(".product-name").filter(`[title="${tshirtName}"]`);
+      
+     }
     public selectTshirt(tshirtName:string) {
-      cy.get(this.tshirts)
-        .find('.product-name')
+        this.findTShirtByName(tshirtName)
         .contains(tshirtName)
         .first()
         .get(this.addToCartButton)
```
Comment (holgiosalos) [inline cypress/page/t-shirts/t-shirts.page.ts]: we can avoid this if you use again filter or find method using the addToCartButton selector

File: cypress/page/t-shirts/t-shirts.page.ts [hunk 4]
```diff
@@ -8,12 +8,19 @@ export class Tshirts{
...
         .click();
     }
+
+
```
Comment (holgiosalos) [inline cypress/page/t-shirts/t-shirts.page.ts]: remove this blank lines

**Review by manuelq12 — CHANGES_REQUESTED** [review]
@cristiancobo Hey cris, the idea of placing the information on the Cypress.json, but as they are global variables is not the right place to put them. If you would like to go with this approach you need to place this information on the "fixtures" folder

Here is some documentation, if you haven't erased the examples that cypress loads when you install it you can check what i mean
https://docs.cypress.io/api/commands/fixture

**Comment by manuelq12** [comment]
@cristiancobo Hey, can you add the mochawesome report please?

**Comment by cristiancobo** [comment]
> @cristiancobo Hey, can you add the mochawesome report please?
@manuelq12 Report is in the commit

**Review by manuelq12 — APPROVED** [review]

**Pull request merged**